corehq/motech/tests/test_public_only_session.py

```diff
@@ -10,8 +10,10 @@
 
 def test_public_only_session__simple_success():
     session = _set_up_session()
-    response = session.get('https://example.com/')
-    eq(response.status_code, 200)
+    with _patch_session_with_hard_coded_response(session, 'https://example.com/',
+                                                 _get_200_response()):
+        response = session.get('https://example.com/')
+        eq(response.status_code, 200)
```

Comment:
Adding to @snopoke's question, what is this testing?

Reply:
Okay I'll add that